src/main/java/de/retest/recheck/report/TestReport.java

```diff
@@ -24,7 +24,7 @@
 public class TestReport extends Persistable {
 
 	private static final long serialVersionUID = 1L;
-	private static final int PERSISTENCE_VERSION = 20;
+	public static final int PERSISTENCE_VERSION = 20;
```

Comment:
Isn't package private sufficient?

Reply:
I use that in `KryoPersistence`, so that is not possible.